src/main/java/io/vertx/mqtt/impl/MqttServerImpl.java

```diff
@@ -85,8 +80,34 @@ public MqttServer listen(Handler<AsyncResult<MqttServer>> listenHandler) {
   public MqttServer listen(int port, String host, Handler<AsyncResult<MqttServer>> listenHandler) {
     Handler<MqttEndpoint> h1 = endpointHandler;
     Handler<Throwable> h2 = exceptionHandler;
-    Handler<MqttConnection> mqttConnectionHandler = c -> c.init(h1, h2);
-    listen(mqttConnectionHandler, port, host, ar -> listenHandler.handle(ar.map(this)));
+    server.connectHandler(so -> {
+      NetSocketInternal soi = (NetSocketInternal) so;
+      soi.messageHandler(o -> {});
+      ChannelPipeline pipeline = soi.channelHandlerContext().pipeline();
+
+/*
+      if (logEnabled) {
+        pipeline.addLast("logging", new LoggingHandler());
+      }
+*/
```

Comment:
it should not be removed, as the NetServerImpl already does it along with other base handlers